File: 251220/중앙값 구하기/find-the-median.js
const fs = require('fs');
const [a, b, c] = fs.readFileSync(0).toString().split(' ').map(v => Number(v));

// a가 가장 클 때
if (a >= b && a >= c) {
    if (b >= c) {
        console.log(b);
    } else {
        console.log(c);
    }
}
// b가 가장 클 때
else if (b >= a && b >= c) {
    if (a >= c) {
        console.log(a);
    } else {
        console.log(c);
    }
}
// c가 가장 클 때
else if (c >= a && c >= b) {
    if (a >= b) {
        console.log(a);
    } else {
        console.log(b);
    }
}

File: 251221/중앙값 구하기/find-the-median.py
a, b, c = map(int, input().split())

# a가 b보다 크거나 같을 때
if a >= b:
    if b >= c:
        print(b)
    else:
        if a <= c:
            print(a)
        else:
            print(c)
# b가 a보다 크거나 같을 때
else:
    if a >= c:
        print(a)
    else:
        if b >= c:
            print(c)
        else:
            print(b)